mmv1/products/workstations/WorkstationConfig.yaml

```
@@ -75,6 +75,11 @@ examples:
       workstation_config_name: 'workstation-config'
       tag_key1: 'tag_key1'
       tag_value1: 'tag_value1'
+      project_id: 'my-new-project'
+      org_id: '123456789'
+    test_vars_overrides:
+      project_id: 'fmt.Sprintf("tf-test-workstation-proj-%s", acctest.RandString(t, 10))'
```

```suggestion
      project_id: 'fmt.Sprintf("tf-test-proj-%s", acctest.RandString(t, 10))'
```

Addressing char limit failures